deepmd/train/trainer.py

```diff
@@ -553,29 +563,31 @@ def train(self, train_data=None, valid_data=None):
         is_first_step = True
         self.cur_batch = cur_batch
         if not self.multi_task_mode:
-            log.info(
-                "start training at lr %.2e (== %.2e), decay_step %d, decay_rate %f, final lr will be %.2e"
-                % (
-                    run_sess(self.sess, self.learning_rate),
-                    self.lr.value(cur_batch),
-                    self.lr.decay_steps_,
-                    self.lr.decay_rate_,
-                    self.lr.value(stop_batch),
-                )
-            )
-        else:
-            for fitting_key in self.fitting:
+            if self.lr_type == "exp":
```

Comment:
It's not a good behavior to switch the learning rate in the `Trainer`. Instead, implement the method `LearningRate.log_start` (`LearningRate` should be an abstract base class and inherited by all learning rate classes) and call `self.lr.log_start(self.sess)` here.